app/src/main/java/com/example/woloxapp/ui/login/LoginFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.woloxapp.ui.login
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.woloxapp.databinding.LoginFragmentBinding
+
+class LoginFragment : Fragment() {
+    private var _binding: LoginFragmentBinding? = null
```

Comment:
this is okay, but if you want to avoid to have two same variables representing the same, take a look to lateinit keyword